test/common/http/http1/codec_impl_test.cc

```diff
@@ -82,6 +84,8 @@ void Http1ServerConnectionImplTest::expectHeadersTest(Protocol p, bool allow_abs
 }
 
 TEST_F(Http1ServerConnectionImplTest, EmptyHeader) {
+  initialize();
```

Comment:
Ah, the whole reason I had to move this from setUp() is one test needs to do work before it's called.

TEST_F(Http1ClientConnectionImplTest, WatermarkTest)  has a mock which causes buffer limits to happen.

I could save a bunch of lines of code by having a subclass test which overrides 
SetUp() {}
initialize { baseclass::SetUp(); }
if we prefer that - I've done it elsewhere and don't have a strong preference for either mode.